Test/Int/ForallExists.expected

```diff
@@ -1,8 +1,7 @@
 goal: ∀ (x : Nat), ∃ y, Int.ofNat x ≤ y
 
 query:
-(define-sort Nat () Int)
-(assert (not (forall ((x Nat)) (=> (>= x 0) (exists ((y Int)) (<= x y))))))
+(assert (not (forall ((x Int)) (=> (>= x 0) (exists ((y Int)) (<= x y))))))
```

Comment:
Output change 1: goals like above are encoded as `Int`s directly. Do you think this is okay or a bug?